Settings/maslowSettings.py

```diff
@@ -215,8 +215,8 @@
             {
                 "type": "bool",
                 "title": "Spindle Automation",
-                "desc": "Should the spindle start and stop automatically based on gcode? Leave off for default stepper control.",
-                "key": "zAxisAuto",
+                "desc": "How should the spindle start and stop automatically based on gcode? Leave off for default external servo control, on for external relay control.",
```

Comment:
I still feel like I wouldn't understand...or maybe I am just thinking the setting does something different.

The options are `off -> The pin toggles digitally high/low to control a relay` or `on -> The pin is a PWM to set speed`...is that right?

Reply:
That's backwards... if the setting is off (the default) AUX1 has a servo signal that is controlled by Mx codes. If the setting is on, the output is LOW or HIGH as would be used to drive an external relay. Maybe 'auxServoRelay'?
 Servo control is different from PWM. Servos, as used here, are motor/gearchain/crank devices that are driven to specific positions of the crank. While the signal is a controlled pulse width, the width of the pulse represents an angle instead of a speed. A 1.5mS pulse will put the servo at its 'neutral' angle, the middle of its range. Varying the pulse width up or down by 1mS will turn the output by 90 degrees in either direction. Removing the signal leaves the output at its present location for some of the simpler examples. That is the way this servo signal is configured, a brief drive to one of the active positions (0 or 180 degrees) followed by a brief 'return to center' pulse train.

Reply:
Woah, thanks for the explanation...that is way different than what I would have expected. 

I like that we have the servo option, but I'm not sure it should be the default...I vote for just regular on off to be the default...but that's not about this PR 😀